db/setupDatabase.js

```diff
@@ -1,23 +1,50 @@
 //set up the database if not exist
-var serverUrl = '127.0.0.1';
-var connection = {
-  host: serverUrl,
-  user: 'root',
-  password: '',
-  charset: 'utf8'
-};
-var knex = require('knex')({
-  client: 'mysql',
-  connection: connection,
-  useNullAsDefault: true
+// This file should be run once from the command line to initialize the db schema.  Be sure to edit the url const with the target url.
+const pg = require('pg');
+
+const url = '<INSERT DB URL HERE FOR SCHEMA UPLOAD>';
+
+var db = require('knex')({
+  client: 'pg',
+  connection: url + '?ssl=true'
+});
+
+db.schema.hasTable('trends').then(function (exists) {
```

Comment:
native method is easier to read but I suppose you and Fufei are partial to this current approach. reference: http://knexjs.org/#Schema-createTableIfNotExists